src/Optimization/UI/AxisymmetricOptimizationUI.jl

```diff
@@ -0,0 +1,546 @@
+# MIT license
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# See LICENSE in the project root for full license information.
+
+module AxisymmetricOptimizationUI
+
+const INTERACTIVE_HTML = false
+
+using OpticSim
+using Blink, WebIO, Interact, Interact.CSSUtil, DataFrames, CSV, TableView
+using ..BlinkUtils
+using ..AxisymmetricOptimization
+
+#
+import Makie
+import Makie.AbstractPlotting
+import Makie.AbstractPlotting.MakieLayout
+if (INTERACTIVE_HTML)
+    import WGLMakie
+else
+    import GLMakie
+end
+
+
+function rg_showtable_sync!(w, schema, names, types, rows, coldefs, tablelength, id, options)
```

Comment:
i'm not sure that prepending your initials onto function/variable names is a good idea - there's a lot of this in the html file too